gunicorn/glogging.py

```diff
@@ -237,7 +237,10 @@ def log(self, lvl, msg, *args, **kwargs):
     def atoms(self, resp, req, environ, request_time):
         """ Gets atoms for log formating.
         """
-        status = resp.status.split(None, 1)[0]
+        if type(resp.status) is int:
```

Comment:
I would do :

status = resp.status
if isinstance(status, str): 
     status = status.split(None, 1)[0]

to be consistent with the style in gunicorn. Thoughts?